### ISSUE TITLE
Powerline/Powerline-Shell/Powerline-go/Powerline-deamon

### ISSUE THREAD

**Issue by ShadabFaiz** [b-ryan/powerline-shell#448]
I tried googling about it, but failed to get any proper answer, so as my last resort, i'm going to try here.
What are the difference between these:
1. Powerline
2. Powerline-Shell
3.Powerline-Go
4.Powerline-deamon.

Are all these same or different variation of the original one( i believe powerline-shell is original)?

**Comment by betafcc** [comment]
TL;DR: If you want to use in a shell (bash, zsh, etc), this repo is the right one 

'Powerline' was just a modeline theme for VIM that became popular and ported to python, I guess this is the original: https://github.com/powerline/powerline

Thou, it is not optimized for use in shell, so this repo became the de facto version of a 'powerline implementation' for use in shell prompt. 'Powerline-daemon' is just an utility on the original version, it's an optional feature to speed up a bit

[powerline-go](https://github.com/justjanne/powerline-go) is an implementation based on 'powerline shell' (this repo), with some features lacking in comparison, but it's faster in general

Another example, [powerline-rs](https://github.com/jD91mZM2/powerline-rs) is another based on 'powerline shell', implemented in rust, it has just a few features, but it's super fast

There are dozens of implementations for a bunch of applications for the concept called 'powerline', for use in shell prompt, this repo is the 'standard'

**Comment by ShadabFaiz** [comment]
Thanks for the info
